classes/PodsUI.php

```diff
@@ -4927,16 +4927,43 @@ public function restricted( $action = 'edit', $row = null ) {
 
 				if ( pods_is_admin( array( 'pods', 'pods_content' ) ) ) {
 					$restricted = false;
-				} elseif ( 'manage' === $action ) {
-					if ( ! in_array( 'edit', $this->actions_disabled ) && ( current_user_can( 'pods_edit_' . $this->pod->pod ) || current_user_can( 'pods_edit_others_' . $this->pod->pod ) ) ) {
-						$restricted = false;
-					} elseif ( ! in_array( 'delete', $this->actions_disabled ) && ( current_user_can( 'pods_delete_' . $this->pod->pod ) || current_user_can( 'pods_delete_others_' . $this->pod->pod ) ) ) {
-						$restricted = false;
-					} elseif ( current_user_can( 'pods_' . $action . '_' . $this->pod->pod ) || current_user_can( 'pods_' . $action . '_others_' . $this->pod->pod ) ) {
-						$restricted = false;
+				} else {
+
+					// Check if the current user is the author of this item.
+					$author = pods( $this->pod->pod, $row )->field( 'author', true );
+					$is_author = false;
+					if ( $author ) {
+						$is_author = (int) $author->ID === (int) wp_get_current_user()->ID;
```

Comment:
* :no_entry_sign: The value of a comparison must not be assigned to a variable
* :no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 7a8145e4f311c3735446b9b49bfef220</sub>